telegram/request/_requestdata.py

```diff
@@ -96,7 +101,13 @@ def parametrized_url(self, url: str, encode_kwargs: Dict[str, Any] = None) -> st
 
     @property
     def json_payload(self) -> bytes:
-        """The parameters as UTF-8 encoded JSON payload."""
+        """The parameters as UTF-8 encoded JSON payload.
+
```

Comment:
```suggestion
        """The :attr:`json_parameters` as UTF-8 encoded JSON payload.
```
?

Reply:
I'll link `parameters` - makes it a bot more clear what's going on IMO :)